nengo_spinnaker/builder/builder.py

```diff
@@ -340,8 +339,10 @@ def make_netlist(self, *args, **kwargs):
                 self, *args, **kwargs
             )
 
-            operator_vertices[op] = vxs
-            vertices.append(vxs)
+            if isinstance(vxs, (list, tuple, set)):
```

Comment:
This is unpleasant - could operators whose ``make_vertices`` method returns a single vertex do so wrapped in a tuple?